src/RokuDeployOptions.ts

```diff
@@ -131,6 +131,11 @@ export interface RokuDeployOptions {
      * If true, the publish will fail on compile error
      */
     failOnCompileError?: boolean;
+
+    /**
+     * Set to true to help with debugging issues in roku-deploy
+     */
+    verboseLogging?: boolean;
```

Comment:
Let's go with `logLevel` instead. I know you only added verbose, and that's fine. I'm ok with having several of the log levels do nothing different. But I don't want to have to change this flag later on. 

```typescript
logLevel?: LogLevel


export enum LogLevel {
    error = 1,
    warn = 2,
    log = 3,
    info = 4,
    debug = 5,
    trace = 6
}
```

Feel free to copy the Logger class from brighterscript: https://github.com/rokucommunity/brighterscript/blob/master/src/Logger.ts

Reply:
resolved. :D